datasets/bbga/bbga.json

```diff
@@ -0,0 +1,204 @@
+{
+  "type": "dataset",
+  "id": "bbga",
+  "title": "BBGA",
+  "status": "beschikbaar",
+  "description": "Basisbestand Gebieden Amsterdam",
+  "version": "0.0.1",
+  "crs": "EPSG:4326",
```

Comment:
Ik vraag mij af of je de CRS weg kan laten aangezien er geen geometrie veld aanwezig is. Het zou kunnen dat de dso-api deze toch wel verwacht. Zouden we moeten testen. Ter info: Bij datasets die wel een geometrie hebben gebruiken we EPSG:28992 ipv 4326

Reply:
Volgens mij klaagde schema-tools zonder "crs". Ik ga het nog even controleren.

Reply:
Ja dat was het. De validator vindt het weglaten van "crs" prima, maar dan loopt de tabel generatie stuk:
```
 File "/usr/home/guido/devel/amsterdam/schema-tools/schematools/types.py", line 20, in __missing__
    raise KeyError(f"No field named '{key}' exists in {self!r}")
KeyError: "No field named 'crs' exists in DatasetSchema
```

@ngaranko had me daarom deze "crs" waarde gegeven.